### ISSUE TITLE
Saving books to txt via bookbot?

### ISSUE THREAD

**Issue by weesnaw-ops** [fr1kin/ForgeHax#132]
How do I save a book to a text file using bookbot? I want to export a book I’ve already written and closed into a text file. Is there any way to do this on 1.12.2 forgehax?

**Comment by LoganDark** [comment]
No, there isn't. You can only save book and quills.

**Comment by weesnaw-ops** [comment]
It's fine. I have an open book, now, I got the admin to open it for me. What do I do now, command-wise? And how do I convert .book to .txt?

**Comment by fr1kin** [comment]
the book file format just has `:PAGE:` delimiters that tell the parser to write to next page. If you want it in text format, just remove those.

**Comment by weesnaw-ops** [comment]
Yes, but how do I save an open book to it. I try .bookbot save, but the .book file is still empty in the end.

**Comment by fr1kin** [comment]
Bookbot doesn't save books, it writes them. I would have to make a separate mod to save them.

**Comment by weesnaw-ops** [comment]
Oh, I see. I thought it did.

**Comment by Mewoco** [comment]
I have a problem where i don't know how to get bookbot to save a txt file to a book.
